### DISCUSSION TITLE
Exclude rule id or ip

### DISCUSSION THREAD

**Discussion by REZAFatf** [Security-Onion-Solutions/securityonion#11420] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

16

### Storage for /

500 gb

### Storage for /nsm

300

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi

How can I exclude (supress) a rule id and ip address from showing on alerts section?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
Have you reviewed the documentation?
https://docs.securityonion.net/en/2.4/managing-alerts.html#so-what-s-next

**Reply by REZAFatf** [reply]
Thank you. How can I suppress only a SID not SID with source or destination ip?

**Reply by dougburks** [reply]
If you want to suppress the SID altogether, have you looked at the `Disable the Alert` section?
https://docs.securityonion.net/en/2.4/managing-alerts.html#disable-the-alert
